march_hardware_interface/src/march_hardware_interface.cpp

```diff
@@ -65,6 +65,15 @@ void MarchHardwareInterface::init()
   joint_position_command_.resize(num_joints_);
   joint_velocity_command_.resize(num_joints_);
   joint_effort_command_.resize(num_joints_);
+  soft_limits_.resize(num_joints_);
+
+  for (int i = 0; i < num_joints_; ++i)
+  {
+    SoftJointLimits soft_limits;
+    getSoftJointLimits(model.getJoint(joint_names_[i]), soft_limits);
+    ROS_INFO("soft_limits_ (%f, %f).", soft_limits.min_position, soft_limits.max_position);
```

Comment:
Do you want this as an ROS_INFO? To pretend spam it could also be a ROS_DEBUG

Reply:
This is in the init method, so this runs only once on start-up.